coral-hive/src/test/java/com/linkedin/coral/hive/hive2rel/HiveToRelConverterTest.java

```diff
@@ -633,6 +633,17 @@ public void testNameSakeColumnNamesShouldGetUniqueIdentifiers() {
     assertEquals(generated, expected);
   }
 
+  @Test
+  public void testCaseInsensitiveFieldAccess() {
+    final String expected = "LogicalProject(s=[$0])\n" + "  LogicalFilter(condition=[=($0.name, 'Jason')])\n"
```

Comment:
This relNode has `condition=[=($0.name, 'Jason')]`. 

what will the hive / spark / trino SQL look like?
Will the generated SQL would have where small-casing for fieldnames in where clause, like - `WHERE view_with_uppercase_struct_field.s.name='Jason'`? or will the fieldName will upper-case?

Reply:
I just removed this unit test as we can't reproduce this issue by unit test because Coral uses Hive 1.2.2 while LI uses 1.1 in production.
But according to my test on the cluster, it would preserve the upper-case field names in the result SQL, so that it's aligned with the base table's schema and the translation could succeed.